sbb_newspapers_org_image/eynollah.py

```diff
@@ -1883,11024 +1276,2424 @@ def do_work_of_slopes_new(self,queue_of_all_params,boxes_text,textline_mask_tot_
-
-
-        peaks_neg=peaks_neg[(peaks_neg>first_nonzero) & (peaks_neg<last_nonzero)]
-        
-        peaks=peaks[(peaks>.09*regions_without_seperators.shape[1]) & (peaks<0.91*regions_without_seperators.shape[1])]
-        
-        peaks_neg=peaks_neg[ (peaks_neg>500) & (peaks_neg< (regions_without_seperators.shape[1]-500) ) ]
-        #print(peaks)
-        interest_pos=z[peaks]
-        
-        interest_pos=interest_pos[interest_pos>10]
-
-        interest_neg=z[peaks_neg]
-        min_peaks_pos=np.mean(interest_pos)#np.min(interest_pos)
-        min_peaks_neg=0#np.min(interest_neg)
-        
-        #$print(min_peaks_pos)
-        dis_talaei=(min_peaks_pos-min_peaks_neg)/multiplier
-        #print(interest_pos)
-        grenze=min_peaks_pos-dis_talaei#np.mean(y[peaks_neg[0]:peaks_neg[len(peaks_neg)-1]])-np.std(y[peaks_neg[0]:peaks_neg[len(peaks_neg)-1]])/2.0
-
-        interest_neg_fin=interest_neg[(interest_neg<grenze)]
-        peaks_neg_fin=peaks_neg[(interest_neg<grenze)]
-
-        num_col=(len(interest_neg_fin))+1
-
-        p_l=0
-        p_u=len(y)-1
-        p_m=int(len(y)/2.)
-        p_g_l=int(len(y)/3.)
-        p_g_u=len(y)-int(len(y)/3.)
-
-        if num_col==3:
-            if (peaks_neg_fin[0]>p_g_u and peaks_neg_fin[1]>p_g_u) or (peaks_neg_fin[0]<p_g_l and peaks_neg_fin[1]<p_g_l ) or (peaks_neg_fin[0]<p_m and peaks_neg_fin[1]<p_m ) or (peaks_neg_fin[0]>p_m and peaks_neg_fin[1]>p_m ):
-                num_col=1
-            else:
-                pass
-
-        if num_col==2:
-            if (peaks_neg_fin[0]>p_g_u) or (peaks_neg_fin[0]<p_g_l):
-                num_col=1
-            else:
-                pass
-
-        diff_peaks=np.abs( np.diff(peaks_neg_fin) )
-        
-
-        cut_off=400
-        peaks_neg_true=[]
-        forest=[]
-        
-        for i in range(len(peaks_neg_fin)):
-            if i==0:
-                forest.append(peaks_neg_fin[i])
-            if i<(len(peaks_neg_fin)-1):
-                if diff_peaks[i]<=cut_off:
-                    forest.append(peaks_neg_fin[i+1])
-                if diff_peaks[i]>cut_off:
-                    #print(forest[np.argmin(z[forest]) ] )
-                    if not self.isNaN(forest[np.argmin(z[forest]) ]):
-                        peaks_neg_true.append(forest[np.argmin(z[forest]) ])
-                    forest=[]
-                    forest.append(peaks_neg_fin[i+1])
-            if i==(len(peaks_neg_fin)-1):
-                #print(print(forest[np.argmin(z[forest]) ] ))
-                if not self.isNaN(forest[np.argmin(z[forest]) ]):
-                    peaks_neg_true.append(forest[np.argmin(z[forest]) ])
-        
-        
-        
-        
-        num_col=(len(peaks_neg_true))+1
-        p_l=0
-        p_u=len(y)-1
-        p_m=int(len(y)/2.)
-        p_quarter=int(len(y)/4.)
-        p_g_l=int(len(y)/3.)
-        p_g_u=len(y)-int(len(y)/3.)
-        
-        p_u_quarter=len(y)-p_quarter
-
-        
```

Comment:
Run codespell before someone else does ;)
(multiple locations)

Reply:
What are you referring to? I supose a comment or print statement but it isn't clear from the diff.

Reply:
There was `rotyate` but I picked the wrong line. Agree that we should check spelling and consistency in a next PR.